integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java

```diff
@@ -87,6 +87,11 @@ public ContainerAllocator(String containerName, int targetNumContainers, int max
     mAllocatedContainers = new ArrayList<>();
   }
 
+  /**
+   * Gets the hosts of workers that could be allocated.
+   *
+   * @return a list of all hosts in the cluster which haven't yet reached the containers per host limit.
```

Comment:
I think this line might be over 100 characters

Also, there can't be a period at the end of `@return`